cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -53,11 +53,12 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   var _isExpanded = false;
   var _isAnimating = false;
 
-  var _expandTransition;
-  var _collapseTransition;
-  var _expandTransitionMethod;
+  var _expandTransition = null;
+  var _expandTransitionMethod = null;
   var _expandTransitionMethodArgs = [];
-  var _collapseTransitionMethod;
+
+  var _collapseTransition = null;
```

Comment:
I think `null` is more appropriate for these. Each variable here is a thing, whether it is a transition or a placeholder for a transition. They're individual things. `undefined` is global singleton essentially and doesn't imply the variable is a placeholder for an object.